src/codegen.cpp

```diff
@@ -8988,8 +8987,8 @@ void jl_compile_workqueue(
                 else {
                     orc::ThreadSafeModule result_m =
                         jl_create_ts_module(name_from_method_instance(codeinst->def),
-                            params.tsctx, params.imaging,
-                            original.getDataLayout(), Triple(original.getTargetTriple()));
```

Comment:
I imagine these are guaranteed to be the same? Or is there a consumer that relies on `params != module`. 
Otherwise LGTM

Reply:
The difference is we don't have to pass in a module anymore (we were constructing a temp module in codegen just to pass along the data layout and target triple).